keyboards/handwired/2x4keypad/readme.md

```diff
@@ -0,0 +1,24 @@
+# 2x4keypad
+
+![2x4keypad](http://jmcameron.net/2x5keypad-small.jpg)
+
+
+This Keypad has 2 rows x 4 columns of keys. Derived from Jonathan Cameron's 2x5.
+It has the top/default layer that has a few handy navigation keys as well as one dedicated key to cycle through
+the layers. The second layer has some media keys. The third and fourth layers
+are currently undefined (although I may use them for inserting accented French
+characters). The keypad also includes three LEDs that show which layer is
+active (no lit LEDs means the default layer).
+
+Keyboard Maintainer: [Kuba Tyszko](https://github.com/kubatyszko)  
+
+Hardware:
+  * 3d-Printed enclosure, STL here: https://www.thingiverse.com/thing:4671269
+  * Handwired
+  * Uses a Teensy
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you include instructions on how to jump into the bootloader?

Reply:
done.